gensim/models/word2vec.py

```diff
@@ -1359,12 +1359,12 @@ def _get_job_params(self, cur_epoch):
         alpha = self.alpha - ((self.alpha - self.min_alpha) * float(cur_epoch) / self.epochs)
         return alpha
 
-    def _update_job_params(self, job_params, epoch_progress, cur_epoch):
+    def _update_alpha(self, alpha, epoch_progress, cur_epoch):
         """Get the correct learning rate for the next iteration.
 
         Parameters
         ----------
-        job_params : dict of (str, obj)
+        alpha : float
```

Comment:
If this is an unused parameter, and the method is internal, perhaps it would be safe to get rid of the parameter?

Reply:
I will include a fix here as well.